birdhouse/README.rst

```diff
@@ -4,9 +4,10 @@
 Docker instructions
 -------------------
 
-Requirements:
+Requirements
+^^^^^^^^^^^^
 
-* Centos 7 or Ubuntu Bionic (18.04), other distros untested.
+* Centos 7, RockyLinux 8, Ubuntu 18.04, 20.04, 22.04, other distros untested.
```

Comment:
Are all of these distros tested as part of a test suite or do we just know they work because we have various deployments running on these? ("other distros untested" implies to me that there are tests for the other distros listed)

Reply:
Fixed wording in 186c3ecb2e14c74fabe3c5cb4ff6344087bcf555.  To explain further, there are no testsuite that continuously test that those distro are still working.

It's basically whatever has been used on our production, staging and on my various test VM at sometime in the past.